components/StreamServer/StreamServer/StreamServer.cpp

```diff
@@ -20,7 +20,8 @@ esp_err_t StreamHelpers::stream(httpd_req_t *req)
   size_t _jpg_buf_len = 0;
   uint8_t *_jpg_buf = nullptr;
 
-  char *part_buf[256];
+  // Buffer for multipart header; was mistakenly declared as array of pointers
+  char part_buf[256];
```

Comment:
One of the mistakes made by an example I followed, never cleaned up, good catch!